pennylane/transforms/classical_jacobian.py

```diff
@@ -40,21 +43,25 @@ def classical_jacobian(qnode):
     >>> @qml.qnode(dev)
     ... def circuit(weights):
     ...     qml.RX(weights[0], wires=0)
-    ...     qml.RY(weights[0], wires=1)
-    ...     qml.RZ(weights[2] ** 2, wires=1)
-    ...     return qml.expval(qml.PauliZ(0))
+    ...     qml.RY(0.2 * weights[0], wires=1)
+    ...     qml.RY(2.5, wires=0)
+    ...     qml.RZ(weights[1] ** 2, wires=1)
+    ...     qml.RX(weights[2], wires=1)
+    ...     return qml.expval(qml.PauliZ(0)@qml.PauliZ(1))

```

Comment:
How come this example was extended?

Reply:
```suggestion
    ...     qml.RY(0.2 * weights[0], wires=1)
    ...     qml.RY(2.5, wires=0)
    ...     qml.RZ(weights[1] ** 2, wires=1)
    ...     qml.RX(weights[2], wires=1)
    ...     return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
```

Reply:
I changed this example for two reasons: To show that more complicated preprocessing than permutation/reusing of parameters is possible and to give an example in which the Jacobian is not square, to make it easier to understand :)